scratch/eyy/test_scripts/profile_band.py

```diff
@@ -102,9 +100,10 @@ def execute(status_dict, func, label, save_dict=True):
 
     # find_freq
     if not args.no_find_freq:
-        subband = None
+        subband = np.arange(10, 120)
```

Comment:
Fine because it's your script but why these subbands?  10,11,12 and 115,116,117,118,119 aren't processed.

Reply:
fixed